java/repositories.bzl

```diff
@@ -112,6 +112,20 @@ def remote_jdk8_repos(name = ""):
         ],
         version = "8",
     )
+    maybe(
+        remote_java_repository,
+        name = "remote_jdk8_macos_aarch64",
+        target_compatible_with = [
+            "@platforms//os:macos",
+            "@platforms//cpu:aarch64",
```

Comment:
Please use `arm64` instead of `aarch64` for names here.

[`@platforms//cpu:aarch64`](https://github.com/bazelbuild/platforms/blob/fbd0d188dac49fbcab3d2876a2113507e6fc68e9/cpu/BUILD#L18) is an alias to [`@platforms//cpu:arm64`](https://github.com/bazelbuild/platforms/blob/fbd0d188dac49fbcab3d2876a2113507e6fc68e9/cpu/BUILD#L63).

Reply:
Done

Reply:
@katre all of the other JDKs are using aarch64, e.g. `remote_jdk8_linux_aarch64` in this file. Can we keep the names consistent to make it easier to automate updates to these, and then separately rename all of them if that's an high priority cleanup?

Reply:
I hadn't realized that.

Yes, let's keep them in sync and clean up all at once. Sorry for the churn.

Reply:
no worries, rolled back